### DISCUSSION TITLE
adonis 5 fresh installation error

### DISCUSSION THREAD

**Discussion by abdullahsayed4131** [adonisjs/core#2131] [Help] [answered]
installing taking too much long time like 5hours.. it is struck at 2 other(prod)
![Capture](https://user-images.githubusercontent.com/37441843/104645468-17aa6d80-56d5-11eb-919a-f57abb75ff66.JPG)

**Comment by thetutlage** [comment]
Impossible to know what is taking 5hours, since it is not something expected. You have to debug yourself and see if your network is able to reach npm or not. Another option to enable the `verbose` mode using following command

```sh
npm init adonis-ts-app first --verbose
```

**Reply by abdullahsayed4131** [reply]
could you please provide me fresh installation of adonis 5 file structure please.....

**Reply by ehsanquddusi** [reply]
Here it is.

![adonis-v5-default-structure](https://user-images.githubusercontent.com/186699/104699422-b5835400-5738-11eb-879a-315d3e98f6ae.jpg)

**Comment by RomainLanz** [comment, marked as answer]
Hey @abdullahsayed4131! 👋 

Do you have `git` installed on your system?

**Reply by abdullahsayed4131** [reply]
sir my issue is cleared by installing node 32 bit insted of 64bit... is there any way to send a array of data from controller to model in adinos js

## in controller

let savingdata = [{

                'name': request.only(['Name']),
                'email': request.only(['Email']),
                'password': request.only(['Password'])
            }];

            let regismodel = new RegistrationModel();
            let result = regismodel.savedata(savingdata);


## in model

export default class RegistrationModel extends BaseModel {

    public async savedata(savingdata)
    {
        const resutlt = await Database
        .insertQuery()
        .table('users')
        .insert(savingdata);
        return resutlt;
    }
}

im getting Cannot read property 'get' of undefined
static $getColumn(name) {
        return this.$columnsDefinitions.get(name);
    }
